internal/registry/registry_cache.go

```diff
@@ -177,11 +180,23 @@ func (r *Cache) CreateCache() error {
 
 	r.RegistryDir = registryDir
 
-	repository, err := git.PlainClone(r.RegistryDir, false, &git.CloneOptions{
-		URL: r.url.String(),
-	})
-	if err != nil {
-		return errors.Wrap(err, "cloning remote registry")
+	if strings.Contains(r.url.String(), "dev.azure.com/") {
```

Comment:
Could this be instead:
```suggestion
	if r.url.Host == "dev.azure.com" {
```